src/Tickets/Admin/Tickets/List_Table.php

```diff
@@ -522,12 +579,23 @@ public function column_end( $item ): string {
 		$date_format = tribe_get_date_format( true );
 		$ts          = $item->end_date();
 
-		return sprintf(
+		$end = sprintf(
 			'<time datetime="%1$s" title="%2$s">%3$s</time>',
 			esc_attr( \Tribe__Date_Utils::reformat( $ts, 'c' ) ),
 			esc_html( \Tribe__Date_Utils::reformat( $ts, get_option( 'date_format' ) . ' ' . get_option( 'time_format' ) ) ),
 			esc_html( \Tribe__Date_Utils::reformat( $ts, $date_format ) )
 		);
+
+		/**
+		 * Filters the end date for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $end The end date for the All Tickets Table.
+		 *
+		 * @return string
+		 */
+		return apply_filters( 'tec_tickets_all_tickets_table_column_end', $end, $item );
```

Comment:
```suggestion
		return apply_filters( 'tec_tickets_all_tickets_table_column_end_date', $end, $item );
```
Let's be clear in the filter that it's the end _date_